### PR TITLE
examiner takes action

### PR DESCRIPTION
**Pull request by Jason-Pullen**
## Description and relevant Jira numbers
mes-77 Examiner must be able to record if a verbal or physical action is required 
## Pull Request checklist:

- [x] [WIP] tag removed from PR title
- [x] PR has an understandable description

## Git feature branch checklist:

- [x] branch name comply with our branching strategy
- [x] git branch contains relevant JIRA ticket number
- [x] branch rebased against the latest develop
- [x] commits are squashed

## Sign off process checklist:

- [x] Code has been tested manually
- [x] Tests are passing
- [x] PR link added to JIRA
- [ ] Reviewers selected in Github
- [] Any new reusable component/widget added to component library on Confluence

- [x] Screenshot(s) captured on the physical device (if applicable)
- [ ] Tested by QA
- [ ] PO's approval
Screenshot 2019-04-04 at 10 53 25
Screenshot 2019-04-04 at 10 53 38
Screenshot 2019-04-04 at 10 53 47

### CODE REVIEW

File: src/pages/test-report/components/examiner-takes-action/__tests__/eta.spec.ts [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import { ComponentFixture, async, TestBed } from '@angular/core/testing';
+import { EtaComponent } from '../eta';
+import { IonicModule } from 'ionic-angular';
+import { testsReducer } from '../../../../../modules/tests/tests.reducer';
+import { StoreModule, Store } from '@ngrx/store';
+import { StoreModel } from '../../../../../shared/models/store.model';
+import { ToggleVerbalEta, TogglePhysicalEta } from '../../../../../modules/tests/test_data/test-data.actions';
+
+describe('LegalRequirementsComponent', () => {
```
Comment (rossfauldsbjss) [inline src/pages/test-report/components/examiner-takes-action/__tests__/eta.spec.ts]: should be ExaminerTakesActionComponent

File: src/pages/test-report/components/examiner-takes-action/__tests__/eta.spec.ts [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        IonicModule,
+        StoreModule.forRoot({ tests: testsReducer }),
+      ],
+    })
+      .compileComponents()
+      .then(() => {
+        fixture = TestBed.createComponent(EtaComponent);
+        component = fixture.componentInstance;
+        store$ = TestBed.get(Store);
+        storeDispatchSpy = spyOn(store$, 'dispatch');
```
Comment (johnmorleybjss) [inline src/pages/test-report/components/examiner-takes-action/__tests__/eta.spec.ts]: No need to assign the return value of of `spyOn`, you can call `expect(store$.dispatch).toHaveBeenCalled()`.

File: src/pages/test-report/components/examiner-takes-action/__tests__/eta.spec.ts (hunk 1 above)
Comment (Jason-Pullen) [reply src/pages/test-report/components/examiner-takes-action/__tests__/eta.spec.ts]: cheers. well spotted

File: src/pages/test-report/components/examiner-takes-action/eta.scss [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    height: 44px;
+    display: flex;
+
+    background: map-get($colors, "mes-white");
+    border: 1px solid rgba(11, 12, 12, 0.4);
+    box-shadow: 0 1px 0 0 map-get($colors-gds, "gds-grey-1");
+    border-radius: 10px;
+    &.complete {
+      background-color:	#F2E8E7;
+      border:2px solid #b10e1e;
```
Comment (johnmorleybjss) [inline src/pages/test-report/components/examiner-takes-action/eta.scss]: This is `gds-red`.

File: src/pages/test-report/components/examiner-takes-action/eta.ts [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+  ngOnInit(): void {
+    const eta$ = this.store$.pipe(
+      select(getTests),
+      select(getCurrentTest),
+      select(getTestData),
+      select(getETA),
+    );
+    this.componentState = {
+      verbal$: eta$.pipe(map(eta => eta.verbal)),
```
Comment (johnmorleybjss) [inline src/pages/test-report/components/examiner-takes-action/eta.ts]: May seem gratuitous, but selectors.

**Pull request merged**